### PR TITLE
chore(functions): move helloAuditLogging to separate directory

### PR DESCRIPTION
**Pull request by ace-n**
Copied from samples in [this directory](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/main/functions/v2).

Those will be deleted once everything is moved over.

Part of b/229394231

### CODE REVIEW

File: functions/v2/helloAuditLog/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "test": "mocha test/index.test.js",
+    "e2e-test": "mocha --timeout 60s test/*.test.js"
```
Comment (averikitsch) [inline functions/v2/helloAuditLog/package.json:16]: do we need both test and e2e-test?

File: .github/workflows/functions-v2-helloauditlog.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+name: functions-v2-helloauditlog
```
Comment (averikitsch) [inline .github/workflows/functions-v2-helloauditlog.html:1]: files extension should be updated to YAML

**Review by averikitsch — APPROVED** [review]

File: functions/v2/helloAuditLog/package.json (hunk 1 above)
Comment (ace-n) [reply functions/v2/helloAuditLog/package.json]: Nope! Removed.

File: .github/workflows/functions-v2-helloauditlog.html (hunk 2 above)
Comment (ace-n) [reply .github/workflows/functions-v2-helloauditlog.html]: Done.

**Review by bcoe — APPROVED** [review]

**Pull request merged**